src/cpp/transport/discovery/DiscoveryServerWindows.cpp

```diff
@@ -176,5 +214,144 @@ bool DiscoveryServerWindows::send_message(OutputPacket&& output_packet)
     return rv;
 }
 
+template<typename EndPoint>
+template<typename T,
+         typename std::enable_if<std::is_same<T, IPv4EndPoint>::value, bool>::type>
+bool DiscoveryServerWindows<EndPoint>::get_interfaces()
```

Comment:
This could be done in a different way using explicit instantiations.

Reply:
Same comment as in Linux case.